documentation/docs/fastly:cache/SimpleCache/purge.mdx

```diff
@@ -0,0 +1,89 @@
+---
+hide_title: false
+hide_table_of_contents: false
+pagination_next: null
+pagination_prev: null
+---
+
+# SimpleCache.purge
+
+purge the entry associated with the key `key` from the cache.
+
+## Syntax
+
+```js
+purge(key, options)
+```
+
+### Parameters
+
+- `key` _: string_
+  - The key to purge from within the cache.
+
+- `options` _: object_
+  - `scope` _: string_
+    - : Where to purge the content from.
+    - Possible values are:
+      - "global" - This will remove the content from all of Fastly.
+      - "pop" - This will remove the content from the POP that contains the currently executing instance.
+  - `headers`
+    - : Any headers you want to add to your request, contained
+      within a `Headers` object or an object literal with `String` values.
+  - `body`
+    - : Any body that you want to add to your request: this can be an `ArrayBuffer`, a `TypedArray`, a `DataView`, a `URLSearchParams`, string object or literal, or a `ReadableStream` object.
+  - `backend` _**Fastly-specific**_
+  - `cacheOverride` _**Fastly-specific**_
+  - `cacheKey` _**Fastly-specific**_
```

Comment:
I don't see these implemented or tested, is it worth continuing to document them?